src/color.rs

```diff
@@ -42,7 +21,7 @@ pub struct RGBA {
     /// The blue component.
     pub blue: u8,
     /// The alpha component.
-    pub alpha: f32,
+    pub alpha: u8,
```

Comment:
If `RGBA` is becoming an internal data format is it not better to just remove it completely?

Reply:
Eventually? Yes, probably. But doing so as part of this PR would have meant a ton more work, so I'd rather leave that for the future.